ListLinked/LinkedList.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace LinkedList
+{
+    public  class LinkedList<T> : IEnumerable<T>
+    {
+        private int position = 0;
+        public Node<T> head;
+
+
+        public void Add(T value)
+        {
+            if (head == null)
+            {
+                head = new Node<T>(value);
+                return;
+            }
+            Node<T> current = head;
+            while (current.next != null)
+            {
+                current = current.next;
+            }
+            current.next = new Node<T>(value);
+
+        }
+
+        public void AddAt(int index, T value)
+        {
+            Node<T> current = head;
+            int head_position = 0;
+            while (head_position < index - 1)
+            {
+                current = current.next;
+                head_position++;
+            }
+            var next = current.next;
+            current.next = new Node<T>(value);
+            current.next.next = next;
+        }
+
+        public int Length()
+        {
+            int length = 1;
+            Node<T> current = head;
+            while (current.next != null)
+            {
+                current = current.next;
+                length++;
+            }
+            return length;
+        }
+
+        public Node<T> RemoveAt(int index)
+        {
+            Node<T> current = head;
+            int head_position = 0;
+            while (head_position < index - 1)
+            {
+                current = current.next;
+                head_position++;
+            }
+            var removed_node = current.next;
+            current.next = current.next.next;
+            return removed_node;
+        }
+
+        public T ElementAt(int index)
+        {
+            Node<T> current = head;
+            int head_position = 0;
+            while (head_position < index)
+            {
+                current = current.next;
+                head_position++;
+            }
+            return current._value;
+        }
```

Comment:
Try to get used to writing the code which has the same structure. An empty line is missed here.